contrib/gitian-keys/keys.txt

```diff
@@ -0,0 +1,28 @@
+152812300785C96444D3334D17565732E08E5E41 Andrew Chow
+E944AE667CF960B1004BC32FCA662BE18B877A60 Andreas Schildbach
+07DF3E57A548CCFB7530709189BBB8663E2E65CE Matt Corallo (BlueMatt)
+912FD3228387123DC97E0E57D5566241A0295FA9 BtcDrak
+C519EBCF3B926298946783EFF6430754120EC2F4 Christian Decker (cdecker)
+F20F56EF6A067F70E8A5C99FFF95FAA971697405 centaur
+C060A6635913D98A3587D7DB1C2491FFEB0EF770 Cory Fields
+BF6273FAEF7CC0BA1F562E50989F6B3048A116B5 Dev Random
+9A1689B60D1B3CCE9262307A2F40A9BF167FBA47 Erik Mossberg (erkmos)
+E777299FC265DD04793070EB944D35F9AC3DB76A Michael Ford
+01CDF4627A3B88AAE4A571C87588242FBE38D3A8 Gavin Andresen
+D3CC177286005BB8FF673294C5242A1AB3936517 jl2012
+D2D1085B9425F9DEFA06E7932270E30C522739F6 Jonas Schnelli
+32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC Jonas Schnelli
+4B4E840451149DD7FB0D633477DFAB5C3108B9A8 Jorge Timon
+71A3B16735405025D447E8F274810B012346C9A6 Wladimir J. van der Laan
+E463A93F5F3117EEDE6C7316BD02942421F4889F Luke Dashjr
+B8B3F1C0E58C15DB6A81D30C3648A882F4316B9B Marco Falke
+CA03882CB1FC067B5D3ACFE4D300116E1C875A3D MeshCollider
+9692B91BBF0E8D34DFD33B1882C5C009628ECF0C Michagogo
+37EC7D7B0A217CDB4B4E007E7FAB114267E4FA04 Peter Todd
+D62A803E27E7F43486035ADBBCD04D8E9CCCAC2A Paul Rabahy
+D762373D24904A3E42F33B08B9A408E71DAAC974 Pieter Wuille (Location: Leuven, Belgium)
+133EAC179436F14A5CF1B794860FEB804E669320 Pieter Wuille
+ED9BDF7AD6A55E232E84524257FF9BDBCC301009 Sjors Provoost
+77E72E69DA7EE0A148C06B21B34821D4944DE5F7 Nils Schneider
+79D00BAC68B56D422F945A8F8E3A8F3247DBCBBF Willy Ko
+AEC1884398647C47413C1C3FB1179EB7347DC10D Warren Togami
```

Comment:
I verified that I was able to retrieve all the other keys, and that the names listed here were correct.